apiclient/harvester_api/managers.py

```diff
@@ -427,7 +434,7 @@ def create_data(self, name, namespace, vlan_id, bridge_name):
                         "serverIPAddr": "",
                         "cidr": "",
                         "gateway": ""
-                    })
+                    }, separators=(',', ':'))
```

Comment:
> If specified, separators should be an (item_separator, key_separator) tuple. The default is (', ', ': ') if indent is None and (',', ': ') otherwise. To get the most compact JSON representation, you should specify (',', ':') to eliminate whitespace. [^1]

It seems we don't have reason to add separators in here ? :thinking: 


[^1]: https://docs.python.org/3/library/json.html#json.dump